src/static/pycontw-2020/styles/page.scss

```diff
@@ -381,7 +381,16 @@ main {
     }
 }
 
+// customize privacy_policy and guidelines page ol
+ol.custom_ol {
+margin-top: 20px;
+}
 
+// customize privacy_policy and guidelines page h5 edited_info
+.edited_info{
+text-align: right;
+color: darkgray;
+}
```

Comment:
nit: I think we can have better naming for `custom_ol` & `edited_info`.

Reply:
I have no idea about naming , so I create a temp name for it . 🤔

Reply:
ok, we can leave it like this for now. at least you put the clear comment :).